src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <!-- <link rel="stylesheet" href="./styles/normalize.css"> -->
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__block-img">
+        <img
+          src="./images/imac.jpeg"
+          alt="Monoblock view"
+          class="card__monoblock-image"
+        >
+      </div>
+      <div class="card__block-text">
+        <h1 class="card__name-monoblock">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="card__product-code">Product code: 195434</p>
+      </div>
+      <div class="card__block-rage stars">
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <div class="stars__item"></div>
+        <p class="rage-text">
+          Reviews: 5
+        </p>
+      </div>
+      <div class="card__block-price">
+        <p class="card__title-price">Price:</p>
+        <h4 class="card__price">$2,199</h4>
+      </div>
+      <div class="card__block-button-buy " >
+        <a
+          href="#"
+          class="link-button"
```

Comment:
same as "rage-text". it is not the element of card or stars, so we cant use it in this block this way